Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -1168,6 +1178,114 @@ function Set-IdentityUri {
     }
 }
 
+function Get-WebDeployPackagePath{
+    param(
+        [Parameter(Mandatory=$true)]
+        [string] $standalonePath,
+        [Parameter(Mandatory=$true)]
+        [string] $installerPath
+    )
+    if(Test-Path $standalonePath){
+        return Resolve-Path $standalonePath
+    }
+    if(Test-Path $installerPath){
+        return Resolve-Path $installerPath
+    }
+    Write-DosMessage -Level "Fatal" -Message "Could not find the web deploy package at $standalonePath or $installerPath."
+}
+
+function Get-WebDeployParameters{
+    param(
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $serviceConfig,
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $commonConfig
+    )
+
+    Confirm-ServiceConfig -commonConfig $commonConfig -serviceConfig $serviceConfig 
+    $metaDataConnectionString = Get-MetadataConnectionString -commonConfig $commonConfig -serviceConfig $serviceConfig
+    
+    $webDeployParameters = @(
```

Comment:
I think we need the DiscoveryServiceEndPoint, UseDiscoveryService, and DomainName parameters in here.